packages/react/src/components/organization-switcher/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useConfiguration } from "../../hooks/use-configuration";
+import { useOrganizations } from "../../hooks/use-organizations"
+import { sprinkles } from "../../theme/sprinkles.css";
+import { Dropdown } from "../dropdown"
+import { Spinner } from "../spinner/spinner";
+
+export function OrganizationSwitcher() {
+  const { text } = useConfiguration();
+  const { organizations, currentOrganization, switchOrganization, isLoading } = useOrganizations()
+
+  if (isLoading || !currentOrganization) return <Spinner /> // need a better loading state
```

Comment:
This is the first time we offer a component that has to load something first - maybe it needs an optional loading fallback prop? Alternatively we could disable the input while it's loading 🤔

Reply:
Sounds great, thank you!